proposals/wgs/overview.md

```diff
@@ -0,0 +1,38 @@
+# Kubeflow Working Group Governance Proposal
+
+## Objective ## 
+
+1. Define the processes for forming and governing Kubeflow working groups.
+1. Push technical and project decision making down to WGs
+1. Push operational responsibilities(releases, testing, etc...) down to WGs
+
+## Background ##
+
+In 2019 the Kubeflow community approved the [Kubeflow Governance Model]. 
+A major pillar of KF's governance model is working groups that own subprojects.
+At the time, formation of WGs and the precise processes by which WGs would operate
+was delayed until KF was large enough to warrant WGs.
+
+KF has now reached a size where forming WGs is critical to sustaining and growing
+the community.
+
+This proposal aims to formalize the governance of WGs. The proposal
+is based on how Kubernetes SIGs operate. In particular, the following
+docs were used as starting points and then adopted to fit Kubeflow:
+
+Original(Kubernetes) | Kubeflow |
+--- | --- |
+[Lifecycle of SIGs and Working Groups](https://github.com/kubernetes/community/blob/master/sig-wg-lifecycle.md) | [Lifecycle](wg-lifcyle.md)
+[SIG Governance](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md) | [WG Governance](wg-governance.md)
+[SIG Charter Guide](https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md) | [WG Charter Guide](wg-charter.md)
+[SIG Governance Requirements](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance-requirements.md) | [WG Governance Requirements](wg-governance-requirements.md)
+[SIG Charter Template](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-charter-template.md) | [WG Charter Template](templates/wg-charter-template.md)
+[Community membership](https://github.com/kubernetes/community/blob/master/community-membership.md) | [Community membership](community-membership.md)
+
+## References
+
+* [Kubeflow Governance Model](https://bit.ly/kf-governance)
+* [KF WG Nominations](https://bit.ly/kubeflow-nominations)
```

Comment:
I am assuming this doc has the nominations and not part of the LGTM of the PR.

Reply:
Correct. As mentioned above it is just a starting point to get people  thinking about what WGs they want, who would be involved and help initiate WG proposals.

The actual process for forming WGs is here.
https://github.com/kubeflow/community/blob/6a876715e28d653cb8fe24ccca816bb72b3ac330/proposals/wgs/wg-lifecycle.md

Reply:
@jlewi we should rename metadata to Model Mangement/Metadata